src/components/ProjectSearch.js

```diff
@@ -178,8 +176,7 @@ export class ProjectSearch extends Component<Props, State> {
   onEnterPress = () => {
     if (
       !this.isProjectSearchEnabled() ||
-      !this.state.focusedItem ||
-      this.state.inputFocused
+      !this.state.focusedItem
```

Comment:
I'm not sure how to trigger this code-path, pressing `Enter` doesn't seem to do it.